packages/components/addon/components/hds/alert/index.js

```diff
@@ -131,10 +133,24 @@ export default class HdsAlertIndexComponent extends Component {
   }
 
   @action
-  didInsertContent(element) {
-    let actions = element.querySelectorAll('button, a');
+  didInsert(element) {
+    let actions = element.querySelectorAll(
+      '.hds-alert__content button,.hds-alert__content a'
+    );
     if (actions.length) {
       this.role = 'alertdialog';
+
+      let title = element.querySelector('.hds-alert__title');
+      if (title) {
+        let titleId = title.getAttribute('id') || guidFor(element);
```

Comment:
@alex-ju do you think we should add also an integration test to make sure that the ID passed to the `Title` is respected/used, when it's provided?